apps/portal-front/src/utils/shareable-resources/shareable-resources.consts.ts

```diff
@@ -60,9 +60,9 @@ function makeSingleQueryMapEntry<TReturn>({
 }
 
 export const localeMap: Record<ServiceSlug, string> = {
-  [ServiceSlug.OPEN_CTI_INTEGRATION_FEEDS]: 'OpenctiIntegrationFeeds',
-  [ServiceSlug.OPEN_CTI_CUSTOM_DASHBOARDS]: 'OpenctiCustomDashboards',
-  [ServiceSlug.OPEN_BAS_SCENARIOS]: 'ObasScenario',
+  [ServiceSlug.OPEN_CTI_INTEGRATION_FEEDS]: 'CsvFeed',
+  [ServiceSlug.OPEN_CTI_CUSTOM_DASHBOARDS]: 'CustomDashboards',
```

Comment:
I don't think this lines must be updated and we should keep `OpenctiIntegrationFeeds` and `OpenctiCustomDashboards`